src/store/actions/index.js

```diff
@@ -0,0 +1,6 @@
+import { EXAMPLE_TYPE } from '../constants';
```

Comment:
We do not need this file right?

Reply:
Yes, we don't actually. I will just go ahead and remove it.